SU-HSDO/suhumsci
# SHS-5505: Banner Image (s) - with partial overlay and text Title is not aligned properly when right aligned

# READY FOR REVIEW

## Summary
- Fix aligment issue in "Banner image with partial overlay and text" component title when right aligned

## Need Review By (Date)
03/08

## Urgency
medium

## Steps to Test
1. Visit a page with a "Banner Image (s) - with partial overlay and text" component (for example https://hs-traditional.suhumsci.loc/)
2. Confirm that at least one of the slides is right aligned. If not, edit the paragraph and change the alignment of one of the slides to "right"
3. Confirm that the title is correctly aligned in all resolutions

<img width="1196" alt="Screenshot 2024-03-04 at 15 23 41" src="https://github.com/SU-HSDO/suhumsci/assets/3496574/bbc7fc0b-e1d0-40df-acff-a959e5e7f475">

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Humsci Basic PR Checklist](https://github.com/SU-HSDO/suhumsci/blob/develop/docs/HumsciBasicPRChecklist.md)
